publisher-sdk/src/main/java/com/criteo/publisher/CriteoInterstitialExt.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ *    Copyright 2020 Criteo
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.criteo.publisher
+
+import com.criteo.publisher.model.InterstitialAdUnit
+
+internal val CriteoInterstitial.adUnit: InterstitialAdUnit?
+    get() = this.interstitialAdUnit
```

Comment:
Why is it preferable to have an extension rather than just exposing a package-private getter in `CriteoInterstitial`?

Reply:
The `InterstitialLogMessage` lives in `com.criteo.publisher.interstitial` package. The `CriteoInterstitial` is in `com.criteo.publisher`. And it cannot be moved to the `interstitial` package because it is a API braking change